plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -149,6 +150,24 @@ private function add_group_blocks() {
 				)
 			);
 		}
+
+		// Linked Products tabgit.
```

Comment:
> tabgit

Is this a typo???

Reply:
Fixed https://github.com/woocommerce/woocommerce/pull/43009/commits/0605d7fef01a53582174152b3de463e1f763e7c0